src/gentropy/dataset/study_locus.py

```diff
@@ -453,6 +457,60 @@ def clump(self: StudyLocus) -> StudyLocus:
         )
         return self
 
+    def get_gnomad_matrix(
```

Comment:
I don't think this method conceptually belongs here. There's a very low-level operations (touching block-matrices), that more suitable for the GnomAD data-source. (you can consider how things would change if more possible LD matrices were added not just GnomAD. If this model of implemenation stays, all subsequent method were expected to be in this same module.

Reply:
Ah ok, originally I thought it might best fit into method/ld.py, but wasn't sure.